scripts/form-editor-support.js

```diff
@@ -23,6 +23,7 @@ import { handleAccordionNavigation } from '../blocks/form/components/accordion/a
 
 window.currentMode = 'preview';
 let activeWizardStep;
+let activeAccordionPanel;
```

Comment:
Since we have made this a global variable, If there are two accordions in the same form, how will this work?
Am i missing something here?

Reply:
The open/closed state of panels in an accordion is stored via the `accordion-collapse` class and not the global variable. `activeAccordionPanel` only keeps track of the currently selected panel in the UE canvas. We need this because when adding a new component, we have a fresh re-render of the accordion and form block does not keep track of the last opened panel, giving a bad UX of a collapsing panel in accordion when a component is added to it in UE.
It works likewise in wizard, a component could be added to a 3rd panel, and without `activeWizardStep`, runtime would, by default, show the first panel in the wizard each time a component is added/a property is changed.

Reply:
ok